source/_components/sensor.elkm1.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Elk-M1 Sensor"
+description: "Instructions on how to integrate Elk-M1 counters, keypads, panel, settings, and zones."
+date: 2018-10-11 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: elkproducts.png
+ha_release: 0.82
```

Comment:
0.81

Reply:
OK. I'm curious now. How would I ever be able to get the version right?

Reply:
@gwww You won't... not until the parent is merged.
A good start is to pick the "next" major version on creation.